liquibase-maven-plugin/src/main/java/org/liquibase/maven/plugins/AbstractLiquibaseMojo.java

```diff
@@ -875,6 +858,28 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         }
     }
 
+    protected Database getDatabase(ClassLoader mavenClassLoader) throws DatabaseException {
```

Comment:
I think this method could be called something like `createDatabase` and some javadoc would be nice. When I saw `getDatabase ` being used in a `try with resources`  in `AbstractLiquibaseUpdateMojo `at first i tough it was a property getter.

Reply:
Yeah, sure. I created this PR yesterday especially to see if there was something broken I could figure out through the functional tests. What you are proposing here makes total sense to me and I'll apply it now. 

Thanks,
Daniel.